docs/devel/README.md

```diff
@@ -415,7 +418,7 @@ make TARGETS=windowmanager subpackages
 ```
 
 Check rpms created in PT Actions under PR Checks > Packit-as-a-Service
-In case new tests need the sub-package, it will be installed immediatly
+In case new tests need the sub-package, it will be installed immediately
 on Packit-as-a-Service test phase.
 
 ### Install PR copr subpackages on local machine
```

Comment:
This one could be deleted also, 
subpackages are not created in copr anymore

Reply:
Removed this piece also. Thanks